automation/01-processForm.js

```diff
@@ -165,6 +165,7 @@ get(`${state.data.url}`, {}, state => {
             type: 'text',
             depth: 0,
             path: [],
+            unique: true,
```

Comment:
@lakhassane , i see how `GeneratedUUID` is set as unique in posgtres (02a) but how is this, or other instances where someone passes in the unique constraint in their list of fields, getting marked as unique in postgres?

Reply:
That could still be correct @taylordowns2000 . I think we can have more than one `uuid` so having another field as `unique` will allow to to upsert on another field than `GeneratedUuid`